src/rust/ide/lib/ast/impl/src/lib.rs

```diff
@@ -852,8 +890,8 @@ impl TokenConsumer for IdMapBuilder {
     fn feed(&mut self, token:Token) {
         match token {
             Token::Off(val) => self.offset += val,
-            Token::Chr( _ ) => self.offset += 1,
-            Token::Str(val) => self.offset += val.len(),
+            Token::Chr(val) => self.offset += val.len_utf8(),
+            Token::Str(val) => self.offset += val.chars().count(),
```

Comment:
Once we add the number of bytes, and then chars count?

Reply:
Stupid mistake. One was wrong and one was correct but I just mindlessly flipped both. Fixed.